xds/src/main/java/io/grpc/xds/ClientXdsClient.java

```diff
@@ -222,79 +234,247 @@ private static LdsUpdate processClientSideListener(Listener listener, boolean pa
       throw new ResourceInvalidException(
           "Could not parse HttpConnectionManager config from ApiListener", e);
     }
+    return LdsUpdate.forApiListener(
```

Comment:
Should the param on line 225 be actually called `parseHttpFilter` (or `parseHttpFilters`) ?

Reply:
You are right, I was about to change that, but then found there are many existing places that use "filter" and `HttpFilter` interchangeably (and even C-core does it). It's not a big problem as they are just local variables. But it's misleading, I will update them.

Reply:
Updated.